### PR TITLE
Improved popup design

### PR DESCRIPTION
**Pull request by dylancom**
Fixed width, smallers texts, better alignment, backgrounds etc.

### CODE REVIEW

File: src/app/router/Popup/styles.scss [hunk 1]
```diff
@@ -1,7 +1,5 @@
 #popup {
-  min-width: 350px;
-  max-width: 540px;
-  min-height: 420px; /* Needed to expand the user menu. */
+  min-height: 420px; /* Needed to account for the user menu. */
```
Comment (bumi) [inline src/app/router/Popup/styles.scss:2]: do we need a max-height?

File: src/app/components/TransactionsTable/index.tsx [hunk 2]
```diff
@@ -24,46 +24,42 @@ type Transaction = {
...
-                          </div>
-                        </Disclosure.Panel>
+                <>
+                  <div key={tx.id} className="flex">
+                    <div className="flex items-center flex-shrink-0 mr-3">
+                      {tx.type && renderIcon(tx.type)}
+                    </div>
+                    <div className="overflow-hidden mr-3">
+                      <div className="text-sm font-medium text-gray-900 truncate">
+                        {tx.title}
```
Comment (bumi) [inline src/app/components/TransactionsTable/index.tsx:58]: where do we set the link for this? can we link directly to the publishers detail page? (right now we link to the publishers list.

but I guess in the parent component where we use the TransactionTable we do not know the link of the individual payment and thus only have one link for all..?

Comment (dylancom) [reply src/app/components/TransactionsTable/index.tsx:58]: We can link to: options.html/publishers/${id}

Comment (bumi) [reply src/app/components/TransactionsTable/index.tsx:58]: ah true, we can do that here? https://github.com/getAlby/lightning-browser-extension/blob/master/src/app/screens/Home/index.jsx#L154 correct?

Comment (dylancom) [reply src/app/components/TransactionsTable/index.tsx:58]: I updated the links for an allowance. When there is no allowance (just payments), I'm not sure yet how we can get the publisher id.

**Pull request merged**